Web_Development/FrontEnd/Html_Accessibility.md

```diff
@@ -0,0 +1,84 @@
+# HTML Accessibility
+This provised the user to navigate and interact in  a better way with your website . That's make your HTML code as <b>senmantic</b> as possible.
+
+## Semantic HTML
+These are the elements which have a meaning or we can say they clearly define about the content which they holding.
```

Comment:
Here too.

Reply:
Done mam now you can check and able to merge also @padmajabhol .

Reply:
Please review It again